### ISSUE TITLE
External CSS

### ISSUE THREAD

**Issue by berryny** [certainignorance/uxc4#1]
Add/Use an external _styles.css_ file.
`<link href="css/styles.css" rel="stylesheet">`

**Comment by berryny** [comment]
Adding a Normalize.css makes browsers render all elements more consistently and in line with modern standards. 
https://necolas.github.io/normalize.css/

**Comment by certainignorance** [comment]
Added external stylesheet and will look at normalize.css

**Comment by certainignorance** [comment]
in my local copy normalize.css is not fix my footer issue (works as intended in chrome). I am looking for a solution now.

**Comment by berryny** [comment]
> in my local copy normalize.css is not fix my footer issue (works as intended in chrome). I am looking for a solution now.

Because styles are all done to `<footer></footer>` add `<footer><div><!--content/tags here --></div></footer>`

**Comment by certainignorance** [comment]
Yeah, that might be the answer. I was hoping to keep it lean but I might have to add divs all the skewed sections to make it consistent.

**Comment by berryny** [comment]
Mobile Hamburger animation example
https://codepen.io/designcouch/pen/Atyop
